config/certmanager/certificate.yaml

```diff
@@ -0,0 +1,26 @@
+# The following manifests contain a self-signed issuer CR and a certificate CR.
```

Comment:
is certmanager stuff mandatory? Do we want to use it?

Reply:
Is not mandatory and not used by default. Kustomize sets is using patches.
The common convention I found following other migrations and their documents is to let the skeleton AS IS and use only what we need.

Reply:
I see. This makes the review harder, however.